src/db/db.js

```diff
@@ -16,13 +16,15 @@ const {
   POSTGRES_USER
 } = require("../config");
 
-const connectionString = `postgres://${POSTGRES_USER}:${POSTGRES_PASS}@${POSTGRES_HOST}:5432/${POSTGRES_DB}?ssl=true`;
```

Comment:
These changes are only for running the db locally - should remove the changes from this file